articles/avoid-fix-flaky-tests.md

```diff
@@ -0,0 +1,73 @@
+---
+authorName: Adrian Herzog
+authorGithubUsername: adiherzog
+issue: 42
+title: How to avoid / deal with flaky system tests
+---
+# {{page.title}}
+
+There are many reasons to avoid automated tests that check an entire system or even a group of connected systems. But there are also good reasons to build and maintain a reasonable number of such tests. For this article, let's assume that the tests are there for a good reason and provide valuable information to the team.
```

Comment:
What are these reasons?  
Examples that come to my mind:
1. Architectural Fitness Functions? 
2. Smoke Tests? 
3. Continuous Health Tests?

Can you link to the other article with those reasons?

Reply:
Rewriting this paragraph. I will avoid writing about why the tests are written and focus on keeping them stable.